plugins/frontend/backstage-plugin-wiz/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@roadiehq/backstage-plugin-wiz",
+  "version": "1.0.0",
```

Comment:
I think this should be 0.0.1 and then bumped by the patchset. Not sure though. Check the other package.json also